packages/client/wallets/smart-wallet/src/types/Config.ts

```diff
@@ -28,7 +28,7 @@ export type PasskeySigner = {
 };
 
 export type EOASigner = EIP1193Provider | ViemAccount;
-export interface WalletConfig {
+export interface WalletParams {
```

Comment:
`config` was getting pretty overloaded, so renaming here. It fits the other dev facing param `UserParams` here better.